packages/embla-carousel-angular/src/components/embla-carousel.directive.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Directive,
+  ElementRef,
+  EventEmitter,
+  Inject,
+  Input,
+  Output,
+  SimpleChanges
+} from '@angular/core'
+import EmblaCarousel, {
+  EmblaCarouselType,
+  EmblaEventType,
+  EmblaOptionsType,
+  EmblaPluginType
+} from 'embla-carousel'
+import {
+  areOptionsEqual,
+  arePluginsEqual,
+  canUseDOM
+} from 'embla-carousel-reactive-utils'
+import { EMBLA_OPTIONS_TOKEN } from '../utils'
+
+@Directive({
+  selector: '[ngx-embla]',
+  exportAs: 'emblaNode',
+  standalone: true
+})
+export class EmblaCarouselDirective {
+  @Input() options: EmblaOptionsType = {}
+  @Input() plugins: EmblaPluginType[] = []
+
+  @Output() emblaEvent = new EventEmitter<EmblaEventType>()
+
+  storedOptions = this.options
+  storedPlugins = this.plugins
+  emblaApi!: EmblaCarouselType
+
+  constructor(
+    public _elementRef: ElementRef<HTMLElement>,
```

Comment:
You could make it `protected`, a public property with a dangling underscore is really uncommon. 
```suggestion
    protected _elementRef: ElementRef<HTMLElement>,
```